core/test/config/filters-test.js

```diff
@@ -552,19 +551,5 @@ describe('Fraggle Rock Config Filtering', () => {
         ],
       });
     });
-
-    it('should preserve full-page-screenshot', async () => {
```

Comment:
Add test for the new flag here?

Reply:
I started on this, but this file is quite more difficult because 1) it doesn't use a real config like the other one does 2) cloning wasn't possible. I had to run before being able to finish, will pick up tomorrow.